robot-core/src/main/java/org/obolibrary/robot/QuotedEntityChecker.java

```diff
@@ -332,28 +311,31 @@ public String getLabel(IRI iri) {
   }
 
   /**
-   * Find an annotation property with the given name. Quotation marks will be removed if necessary.
+   * Find an annotation property with the given name, or create one. Quotation marks will be removed
+   * if necessary.
    *
    * @param name the name of the entity to find
    * @return an annotation property, or null
    */
+  @Override
   public OWLAnnotationProperty getOWLAnnotationProperty(@Nonnull String name) {
-    return getOWLAnnotationProperty(name, false);
+    return getOWLAnnotationProperty(name, true);
   }
 
   /**
-   * Find an annotation property with the given name. Quotation marks will be removed if necessary.
+   * Find an annotation property with the given name, or create one. Quotation marks will be removed
+   * if necessary.
    *
    * @param name the name of the entity to find
-   * @param create when true and an IOHelper is defined, create the property
+   * @param create if false, do not create a new annotation property
```

Comment:
I don't understand this change. Is it just the wording, or has the code changed?

Reply:
Just the wording has changed. I can change it back if the first way makes more sense.